packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -476,9 +485,9 @@ export class WorkspaceCommandContribution implements CommandContribution {
         }
     }
 
-    protected areMultipleOpenHandlersPresent(openers: OpenHandler[], uri: URI): boolean {
+    protected areMultipleOpenHandlersPresent(uri: URI): boolean {
```

Comment:
I'm not sure if such updates to method signatures should be documented in the changelog, its a breaking change for anyone currently calling `areMultipleOpenHandlersPresent`. The same should be confirmed for any API update in the changeset.

Reply:
You are correct so reverted back to original api - as it wasn't really required here.